apps/cowswap-frontend/src/common/updaters/FeesUpdater.ts

```diff
@@ -26,7 +26,7 @@ import { getPriceQuality } from 'api/gnosisProtocol/api'
 import { useVerifiedQuotesEnabled } from '../hooks/featureFlags/useVerifiedQuotesEnabled'
 
 export const TYPED_VALUE_DEBOUNCE_TIME = 350
-const REFETCH_CHECK_INTERVAL = 10000 // Every 10s
+export const SWAP_QUOTE_CHECK_INTERVAL = 10000 // Every 10s
```

Comment:
should we use `ms` macro?

Reply:
Fixed: https://github.com/cowprotocol/cowswap/pull/3756/commits/d23b832308103c9f3a5fde31069618209e3373d6